framework_tvb/tvb/core/entities/storage/datatype_dao.py

```diff
@@ -393,6 +397,8 @@ def get_datatype_by_gid(self, gid, load_lazy=True):
                 result_dt.parent_operation.algorithm.algorithm_category
                 result_dt.parent_operation.operation_group
                 result_dt._parent_burst
+                if load_lazy_extra_info:
```

Comment:
Since we are using another flag for the extra info I think it would be better to move the "if load_lazy_extra_info" outside of the "if load_lazy" statement and when we call the get_extra_info method set just the load_lazy_extra_info flag on True.